container/20230626.md

```diff
@@ -0,0 +1,65 @@
+# 2023-06-26
+## Participants (github handles preferred)
+- @master-caster
+- @o-otte
+- @batistein
+- @matfechner
+- @maxwolfs
+- @mxmxchere
+- @jnull
+- @fkr
+- @ajfriesen
+- @guettli
+- @joshmue
+- @chess-knight
+- @jschoone
+- @saschascherrer
+- @matofeder
+- @janiskemper
+- @bitkeks
+- @wieneo
+- @aniruddha
+- @nitisht
+
+## Community call and minutes transfer
+  - @jschoone
+
+
+## Welcome
+- Aniruddha (Syself), Cloud Engineer and Cluster API maintainer
+- Nitish (Syself), expert in writing operators and cloud-native software
+- Johann Gnaucke, PSKE Architect (Gardener), will replace Sascha in SCS specific topics at PlusServer
+
+## Agenda
+Sprint Review (Edmonton)
+Sprint Planning (Freiburg)
+
+### Dedicated chat room
+- We now have a room on Matrix, see [SCS | Container](https://matrix.to/#/!NZpJdPGjAHISXnHUil:matrix.org?via=matrix.org)
+
+### Sprint review (Edmonton)
+
+#### Ref. Impl.
+
+- Use tf-1.3.6 for github workflow. [k8s/#426](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/426)
+- Update CAPI to v1.3.8 and CAPO to 0.7.3. [k8s/#427](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/427)
+- Update to latest k8s and latest occm versions. [k8s/#428](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/428)
+- Feat/calico 3.25.1 [k8s/#429](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/429)
+- Feat/cilium 1.13.3 [k8s/#430](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/430)
+- Update default flux2 to 0.41.2 [k8s/#435](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/435)
+- Update default cert-manager to 1.12.2 [k8s/#437](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/437)
+- Update helm to 3.12.1 [k8s/#439](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/439)
+
+
+#### Cluster Stacks v0.1: Design Considerations, PoC, ...
+
+- Cluster Stack Example Docker [issues/#348](https://github.com/SovereignCloudStack/issues/issues/348)
+- Document how to implement Cluster Stacks [issues/349](https://github.com/SovereignCloudStack/issues/issues/349)
+- Provide example cluster stacks [issues/#322](https://github.com/SovereignCloudStack/issues/issues/322)
+- How to contribute?
+- Evaluating KubeVirt
+
+##### Lightning Talk about ClusterStacks next week (2023-07-03)
+
+### Lightning talk Cilium
```

Comment:
@jschoone Lightning Talk about ClusterStacks next week (2023-07-03) ? Something is wrong here, or do you want me to give this talk a second time?

Reply:
No, thats the talk about the ClusterStacks from Janis afaik